mempool/src/core_mempool/transaction_store.rs

```diff
@@ -580,16 +595,27 @@ impl TransactionStore {
                     if batch_total_bytes.saturating_add(transaction_bytes) > self.max_batch_bytes {
                         break; // The batch is full
                     } else {
-                        batch.push(txn.txn.clone());
+                        batch.push((txn.txn.clone(), txn.insertion_info.insertion_time));
```

Comment:
I think `ready_time` would be a better metric to use here. If we have things stuck in parking lot, we don't want it attributed to broadcast time

Reply:
As the goal is to measure end-to-end latency of the transaction, we'll have to include even the time the transaction is in parking lot right?

Reply:
Even earlier, we've been calculating (block_timestamp - insertion time) for PFNs and VFNs and subtracting these times to get PFN -> VFN txn latency.

Reply:
The parking lot time is tracked in `"parked_time"`

Reply:
So, should I split the e2e txn latencies plot to "PFN parked time", "PFN -> VFN txn", "VFN parked time", "VFN -> Validator txn"?